wallet/walletrpc.c

```diff
@@ -83,7 +83,7 @@ static void wallet_withdrawal_broadcast(struct bitcoind *bitcoind UNUSED,
  * the HSM to generate the signatures.
  */
 static void json_withdraw(struct command *cmd,
-			      const char *buffer, const jsmntok_t *params)
+			  const char *buffer, const jsmntok_t *params)
```

Comment:
`const` should align with `struct` on previous line, unless this is an artifact of the `+` in the diff, in which case, ignore this comment.

Reply:
Yes. This is an artifact of the + in the diff.